### PR TITLE
Solution to following issues:

### PR DESCRIPTION
**Pull request by Zacorich**
https://github.com/otros-systems/otrosvfsbrowser/issues/11
https://github.com/otros-systems/otroslogviewer/issues/356
https://github.com/otros-systems/otroslogviewer/issues/330

### CODE REVIEW

**Comment by Zacorich** [comment]
As **otroslogviewer** project uses **Commons-libs** project that uses **otrosvfsbrowser** project, next version of otrosvfsbrowser jar needs to go into something like "Common-libs/lib/OtrosVfsBrowser-1.0.1.jar" and finally run Gradle task "**otroslogviewer:OtrosAppDist [dist]**" to create new version of logviewer.

File: src/main/java/pl/otros/vfs/browser/preview/PreviewListener.java [hunk 1]
```diff
@@ -106,30 +106,31 @@ protected PreviewStatus doInBackground() throws Exception {
             return new PreviewStatus(State.CANCELLED, 0, previewLimit / 1024, KB, name, EMPTY_BYTE_ARRAY);
           }
         }
-        ByteArrayOutputStream outputStream = new ByteArrayOutputStream(previewLimit);
-        InputStream inputStream = null;
+        ByteArrayOutputStream outputStreamRef = null;
```
Comment (Zacorich) [inline src/main/java/pl/otros/vfs/browser/preview/PreviewListener.java]: Neede due to the scope of try-with-resources as used later for more general exception

**Comment by Zacorich** [comment]
Check latest commit

**Comment by Zacorich** [comment]
*noticed that you use code style "Spaces within array initializer braces"

File: src/main/java/pl/otros/vfs/browser/preview/PreviewListener.java [hunk 2]
```diff
@@ -105,26 +106,31 @@ protected PreviewStatus doInBackground() throws Exception {
...
-            publish(new PreviewStatus(State.LOADING, outputStream.size() / 1024, max / 1024, KB, name, outputStream.toByteArray()));
           }
         } catch (Exception e) {
           LOGGER.error("Exception when downloading preview", e);
-          return new PreviewStatus(State.ERROR, outputStream.size() / 1024, outputStream.size() / 1024, KB, name, outputStream.toByteArray());
+          return new PreviewStatus(State.ERROR, outputStreamRef.size() / 1024, outputStreamRef.size() / 1024, KB, name, outputStreamRef.toByteArray());
         }
 
-        return new PreviewStatus(State.FINISHED, outputStream.size() / 1024, outputStream.size() / 1024, KB, name, outputStream.toByteArray());
+        return new PreviewStatus(State.FINISHED, outputStreamRef.size() / 1024, outputStreamRef.size() / 1024, KB, name, outputStreamRef.toByteArray());
```
Comment (otros-systems) [inline src/main/java/pl/otros/vfs/browser/preview/PreviewListener.java]: It should go as last call in try, before catch. In this way we can get rid off outputStremRef.

Comment (Zacorich) [inline src/main/java/pl/otros/vfs/browser/preview/PreviewListener.java]: How will you catch State.ERROR then?

Comment (Zacorich) [inline src/main/java/pl/otros/vfs/browser/preview/PreviewListener.java]: I can do something like 
`int outputStreamSize = 0;
        byte[] outputStreamByteArr = new byte[0];
        try (
          ByteArrayOutputStream outputStream = new ByteArrayOutputStream(previewLimit);
          InputStream inputStream = fileObjectToPreview.getContent().getInputStream()
        ) {...} catch (Exception e) {
          LOGGER.error("Exception when downloading preview", e);
          return new PreviewStatus(State.ERROR, outputStreamSize / 1024, outputStreamSize / 1024, KB, name,
            outputStreamByteArr);
        }

        return new PreviewStatus(State.FINISHED, outputStreamSize / 1024, outputStreamSize / 1024, KB, name,
          outputStreamByteArr);`

Comment (Zacorich) [inline src/main/java/pl/otros/vfs/browser/preview/PreviewListener.java]: Another alternative is stop using outputStream in new PreviewStatus() object creation, but then we might lose some feature...

**Comment by otros-systems** [comment]
@Zacorich Thanks for merge request!

**Pull request merged**